h/config.py

```diff
@@ -100,6 +100,7 @@ def configure(environ=None, settings=None):
     # *not* toggle functionality based on this value. It is intended as a
     # label only.
     settings_manager.set("h.env", "ENV")
+    settings_manager.set("h.es_sync_job_limit", "ES_SYNC_JOB_LIMIT", default=1000)
```

Comment:
So we'll have two ways to tune the speed of reindexing in production:

1. We can change the `ES_SYNC_JOB_LIMIT`, which is how many annotations the service will check per run
2. We can change the schedule in h-periodic, which is how frequently the service gets run